doc/source/analyzing/fields.rst

```diff
@@ -396,6 +400,7 @@ aliases were removed in yt v4.1.0.
 
 Magnetic Fields
 ---------------
+TODO: clarify if this section needs an update after https://github.com/yt-project/yt/pull/3812
```

Comment:
@jzuhone Can you please proof read this paragraph ? You wrote it 7 years ago, does your recent PR would require this be updated ?

Reply:
@jzuhone reminder

Reply:
After this gets merged I can supply language in a separate PR

Reply:
Sorry I don't understand what you mean by "supply language"

Reply:
Should I just drop the comment, open an issue and assign it to you ?

Reply:
(this is #4113)